tigerpath/majors_and_certificates/majors/EEB_2018.json

```
@@ -97,113 +124,117 @@
           "completed_by_semester": 7,
           "course_list": [
             "SML 201",
-            "ECO 202",
             "ORF 245",
-            "POL 345",
             "PSY 251",
+            "POL 345",
+            "SOC 301",
+            "ECO 202",
+            "WWS 220",

```

This course doesn't exist. I think it's a typo and they meant WWS 200, which is "Statistics for Social Science"
```suggestion
            "WWS 200",
```
I mean, we could check with the department if you really want to be sure.